cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -182,16 +214,19 @@
     },
     "keywords": {
       "items": {
+        "description": "Relevant keywords for the record",
         "type": "string"
       },
       "type": "array"
     },
     "language": {
+      "description": "The language/s of the resource, based on ISO 639-2",
```

Comment:
Do we allow more than one language? I don't think we do, it is a string in the schema, hence singular is enough.